src/components/ProjectForm.vue

```diff
@@ -0,0 +1,247 @@
+<script>
+import { every, cloneDeep, kebabCase } from 'lodash'
+
+import InlineDirectoryPicker from '@/components/InlineDirectoryPicker'
+import { slugger, isUrl } from '@/utils/strings'
+
+/**
+ * Build project form (to create or edit a project).
+ */
+export default {
+  name: 'ProjectForm',
+  components: {
+    InlineDirectoryPicker
+  },
+  props: {
+    /**
+     * Display the form in a card
+     */
+    card: {
```

Comment:
Is it possible to add default values or required property when necessary for the props of this component? Especially for Object type

Reply:
You mean like typed object in Typescript? Here I used a plain and empty object because every properties are optional

Reply:
Ah yes, you're right. CHanging!